### PR TITLE
#13584252 grunt to gulp taskrunner

### PR DESCRIPTION
**Pull request by collins-b**
#### What does this PR do?
Convert grunt to gulp taskrunner
 #### Description of Task to be completed?
Make sure dotenv is working well
 #### How should this be manually tested?
While in the project root folder in the terminal, type **_gulp_** and press Enter
 #### Any background context you want to provide?
Libraries in the gulpfile.js needs to be installed before running the project
 #### What are the relevant pivotal tracker stories?
#13584252

### CODE REVIEW

**Comment by gngchrs** [comment]
The deployment failed @andela-cwekesa. Check the Heroku Build logs from the deployment and fix them.

**Review by boswellgathu — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,7 @@ public/lib
 .DS_store
 c4h_keys.txt
 .env
+<<<<<<< HEAD
```
Comment (gngchrs) [inline .gitignore]: Why do we have merge conflicts meta here??

Comment (collins-b) [reply .gitignore]: I resolved it.

Comment (gngchrs) [reply .gitignore]: Yes, but the `<<<<<<<Head ` should not be here.

**Comment by gngchrs** [comment]
Please resolve your conflicts

**Pull request closed without merging**